### PR TITLE
Feature 5 : Integrated All PUT APIs

### PR DESCRIPTION
**Pull request by Abhishek-Jain-1925**
Feature 5 : Integrate All PUT APIs

Workdone are as follows - 
1. Integrate All PUT APIs
2. Added primary validations to all forms 
3. Removed unused code by resolving conflicts as well
4. Some UI optimizations

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,10 @@
     "antd": "^5.17.2",
     "axios": "^1.7.2",
     "bootstrap": "^5.3.3",
+    "docx": "^8.5.0",
+    "file-saver": "^2.0.5",
     "http": "^0.0.1-security",
+    "jspdf": "^2.5.1",
```
Comment (Payal13) [inline package.json:22]: Why do we need these packages? remove if not needed

File: src/components/Builder/Achievement/index.jsx [hunk 2]
```diff
@@ -20,39 +23,45 @@ import {
...
+  const [updateAchievementService] = useUpdateAchievementMutation();
   const [form] = Form.useForm();
   const [activeKey, setActiveKey] = useState("0");
   const [items, setItems] = useState([
-    { label: "Achievement 1", children: null, key: "0" }
+    {
+      label: "Achievement 1",
+      children: null,
+      key: "0",
+      isExisting: "",
```
Comment (Payal13) [inline src/components/Builder/Achievement/index.jsx:39]: default value should be false for `isExisting`

File: src/components/Builder/Achievement/index.jsx [hunk 3]
```diff
@@ -92,10 +93,46 @@ const Achievement = ({ achievementData }) => {
...
+    try {
+      for (const achievement of values) {
+        if (achievement.id) {
+          const response = await updateAchievementService({
+            profile_id: profile_id,
+            achievement_id: achievement.id,
+            values: achievement,
+          });
+          if (response.data?.message) {
+            toast.success(response.data?.message);
```
Comment (Payal13) [inline src/components/Builder/Achievement/index.jsx:106]: why we need toast here ? interceptor toaster will not help here ?

File: src/components/Builder/Achievement/index.jsx [hunk 4]
```diff
@@ -92,10 +93,46 @@ const Achievement = ({ achievementData }) => {
...
+          }
+        }
+      }
+    } catch (error) {
+      toast.error(error.response?.data?.error_message);
```
Comment (Payal13) [inline src/components/Builder/Achievement/index.jsx:111]: same here ?

File: src/components/Builder/Education/index.jsx [hunk 5]
```diff
@@ -19,26 +22,25 @@ import {
...
   const [form] = Form.useForm();
   const [activeKey, setActiveKey] = useState("0");
   const [items, setItems] = useState([
     {
       label: "Education 1",
       children: null,
-      key: "0"
-    }
+      key: "0",
+      isExisting: "",
```
Comment (Payal13) [inline src/components/Builder/Education/index.jsx:35]: change default value

File: package.json (hunk 1 above)
Comment (Abhishek-Jain-1925) [reply package.json:22]: Done!

File: src/components/Builder/Achievement/index.jsx (hunk 2 above)
Comment (Abhishek-Jain-1925) [reply src/components/Builder/Achievement/index.jsx:39]: Done!

File: src/components/Builder/Education/index.jsx (hunk 5 above)
Comment (Abhishek-Jain-1925) [reply src/components/Builder/Education/index.jsx:35]: Done!

File: src/components/Builder/Achievement/index.jsx (hunk 3 above)
Comment (Abhishek-Jain-1925) [reply src/components/Builder/Achievement/index.jsx:106]: As per discussion, kept as it is

File: src/components/Builder/Achievement/index.jsx (hunk 4 above)
Comment (Abhishek-Jain-1925) [reply src/components/Builder/Achievement/index.jsx:111]: As per discussion, kept as it is

**Pull request merged**